frameworks/elastic/tests/test_tls.py

```diff
@@ -30,46 +33,69 @@ def service_account(configure_security):
 
 @pytest.fixture(scope='module')
 def elastic_service_tls(service_account):
-    sdk_install.install(
-        config.PACKAGE_NAME,
-        service_name=config.SERVICE_NAME,
-        expected_running_tasks=config.DEFAULT_TASK_COUNT,
-        additional_options={
-            "service": {
-                "service_account_secret": service_account["name"],
-                "service_account": service_account["secret"],
-                "security": {
-                    "transport_encryption": {
-                        "enabled": True
+    try:
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            service_name=config.SERVICE_NAME,
+            expected_running_tasks=config.DEFAULT_TASK_COUNT,
+            additional_options={
+                "service": {
+                    "service_account_secret": service_account["name"],
```

Comment:
These two fields are the swapped.

Reply:
Good catch. I'll fix them here too: https://github.com/mesosphere/dcos-commons/blob/a6b8f0c1b61f37bae4e7dc071a0aff30a6cacbc6/frameworks/hdfs/tests/test_tls.py#L45-L46

Reply:
https://github.com/mesosphere/dcos-commons/pull/2349/commits/ea0aa2e401f8b275f78577ed3dca99af999f1e0a